backend-shared/config/db/guardrow_test.go

```diff
@@ -0,0 +1,890 @@
+package db_test
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	db "github.com/redhat-appstudio/managed-gitops/backend-shared/config/db"
+)
+
+var _ = Describe("GuardRow test", func() {
```

Comment:
Can change 'guardrow test' to something like 'verify update/delete operations are not globally scoped'